files/bridgesi/sprint-1-1_18inchPipePile_bridgeSI.ifc

```diff
@@ -0,0 +1,101 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION ((''), '2;1');
+FILE_NAME ('', '2020-09-17T00:38:34', (''), (''), '', '', '');
+FILE_SCHEMA (('IFC4'));
+ENDSEC;
+DATA;
+
+#1=IFCPROJECT('3vv0vLSb9CVfUYZamBGQvP', $,'Bridge Model',$,$,$,$,(#19,#22),#7);
+
+#2=IFCRELDECLARES('2DAvEupIz0HQr73cMacwtY', $, $, $, #1, (#3));
+#3=IFCACTOR('2DAvEupIz0HQr73cMacwtX', $, $, $, $, #4);
+#4=IFCPERSON($, 'Alexander', 'Shishlov', $, $, $, #5, $);
+#5=IFCACTORROLE(.ENGINEER., $, $);
+
+#7=IFCUNITASSIGNMENT((#8,#9,#10,#11,#12,#13,#14,#15));
+#8=IFCSIUNIT(*,.LENGTHUNIT.,$,.METRE.);
+#9=IFCSIUNIT(*,.AREAUNIT.,$,.SQUARE_METRE.);
+#10=IFCSIUNIT(*,.VOLUMEUNIT.,$,.CUBIC_METRE.);
+#11=IFCSIUNIT(*,.SOLIDANGLEUNIT.,$,.STERADIAN.);
+#12=IFCSIUNIT(*,.PLANEANGLEUNIT.,$,.RADIAN.);
+#13=IFCSIUNIT(*,.MASSUNIT.,$,.GRAM.);
+#14=IFCSIUNIT(*,.TIMEUNIT.,$,.SECOND.);
+#15=IFCSIUNIT(*,.THERMODYNAMICTEMPERATUREUNIT.,$,.DEGREE_CELSIUS.);
+
+#17=IFCCARTESIANPOINT((0.,0.,0.));
+#18=IFCAXIS2PLACEMENT3D(#17,$,$);
+#19=IFCGEOMETRICREPRESENTATIONCONTEXT('Building Model','Model',3,1.E-05,#18,$);
```

Comment:
I think that the issue is that the first attribute should be '3D' according to the concept usage.

Reply:
Much appreciated, have resolved this by editing #19.